index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
```

Comment:
Come up with a way to change the color for each piece, each time the keyboard is pressed to change the shape.